client/scripts/views/pages/landing/creators_card_section.ts

```diff
@@ -10,29 +10,139 @@ const TokensCreatorComponent: m.Component<{ creators: ICardListItem[] }, {}> = {
   view: (vnode) => {
     const { creators } = vnode.attrs;
 
-    return m('section.TokensCreatorComponent', { class: 'container mx-auto pt-10' }, [
-      m(
-        'h2',
-        { class: 'text-3xl font-bold mb-5 text-center' },
-        ' Token creators are empowered '
-      ),
-      m(
-        'p',
-        { class: 'text-2xl max-w-screen-sm mx-auto text-center mb-10' },
-        ' Commonwealth lets you simplify your community and governance, bringing four tools into one. '
-      ),
+    return m('section.TokensCreatorComponent', { class: '' }, [
+      m('div', { class: 'container mx-auto pt-10' }, [
+        m(
+          'h2',
+          { class: 'text-3xl font-bold mb-5 text-center' },
+          ' Token creators are empowered '
+        ),
```

Comment:
do you mind going through these files and linting them? 
for example 15 through 19 here should look like: 
```
m('h2', {
  class: 'text-3xl font-bold mb-5 text-center',
  },  'Token creators are empowered'),
```
also all of the string labels seem to have an extra space prefix and postfix. Those should be removed.